lib/view.js

```diff
@@ -73,7 +73,8 @@ export default class AdvancedOpenFileView {
         return dom(`
             <div class="advanced-open-file">
                 <p class="info-message icon icon-file-add">
-                    Enter the path for the file to open or create.
+                    <span class="default-info-message">Enter the path for the file to open or create.</span>
+                    <span class="prompt-confirmation-message">File does not exist; hit [Enter] again to create it.</span>
```

Comment:
The current implementation is toggling both messages using `style.display = 'none' / 'inline'`.
The benefit is that only style changes, no dom element.
 
Please let me know if there's a better way to achieve this.

Reply:
You could have the message and class be properties, and have a `setMessage` function:

```js
return dom(`
  <span class="${infoMessageClassName}">${infoMessage}</span>
`);
```

```js
setInfoMessage(message, className='default-info-message') {
  this.infoMessage = message;
  this.infoMessageClassName = className;
}
```